specs/_features/eip7251/fork.md

```diff
@@ -128,6 +128,23 @@ def upgrade_to_eip7251(pre: deneb.BeaconState) -> BeaconState:
         pending_consolidations=[],
     )
 
+    activation_queue = []
+    pre_activation_queue = []
+    below_minimum = []
+    for index, validator in enumerate(post.validators):
+        if validator.activation_epoch == FAR_FUTURE_EPOCH:
+            if validator.activation_eligibility_epoch != FAR_FUTURE_EPOCH:
+                activation_queue.append(index)
+            else:
+                if validator.effective_balance >= MIN_ACTIVATION_BALANCE:
+                    pre_activation_queue.append(index)
+                else:
+                    below_minimum.append(index)
+
+    activation_queue.sort(key=lambda index: (post.validators[index].activation_eligibility_epoch, index))
+    for index in activation_queue + pre_activation_queue + below_minimum:
```

Comment:
Good suggestion! I've updated the code. I also removed the secondary sorting condition. My original intention was to order the validators by:
 - validators with `activation_eligibility_epoch` already set
 - validators where `activation_eligibility_epoch` *could* be set # this is the second group
 - validators with `activation_eligibility_epoch` unset

but since `process_epoch()` runs before this, the second group is absorbed into the first group anyway so we don't need that sorting condition.